.github/workflows/release-enigma-go.yml

```diff
@@ -0,0 +1,24 @@
+name: release-enigma-go
+on: 
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Release Version (major, minor, patch)
+        required: true
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Go environment
+        uses: actions/setup-go@v2.1.3
+        with:
+          go-version: '^1.17'
+      - name: Checkout enigma-go repo
+        uses: actions/checkout@v2  
+        with:
+          repository: qlik-oss/enigma-go
+          ref: master
+          fetch-depth: 0
+      - name: Create enigma-go release
+        run: |                  
+          ./release/release.sh ${{ github.event.inputs.version }}
```

Comment:
If I'm not mistaken the release script actually doesn't push the commits, it just the creates the tags and commits. So if this should do the full release you probably need to update the script or add `git push --follow-tags` after the script has been run.

Reply:
Thanks 👍